config/environments/production.rb

```diff
@@ -19,7 +19,7 @@
   config.public_file_server.enabled = false
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = Uglifier.new(harmony: true)
```

Comment:
I don't see any ES6-specific code in this PR,
and not entirely sure how this change would really affect production,
therefore I suggest doing it in a separate PR in case we will need to revert this code.

Reply:
after discussing with @nofaralfasi, seems that the default parameter is an ES6 feature, let's try to avoid using it in this PR and create another PR for the `Uglifier`

Reply:
As we discussed, I created this [PR](https://github.com/theforeman/foreman/pull/9567) for enabling harmony mode with Uglifier.